web/client/themes/dark/variables.less

```diff
@@ -1,5 +1,6 @@
 @ms-main-color: #eeeeee;
 @ms-main-bg: #333333;
+@ms-mask-bg: #00000088;
```

Comment:
This variable is not used please remove it

Reply:
it's used